{{cookiecutter.project_slug}}/README.md

```diff
@@ -4,6 +4,10 @@
 
 ## Setup
 
+### Github & Heroku
+1. [Generate an auth token for Heroku](https://devcenter.heroku.com/articles/heroku-cli-commands#heroku-authorizations-create) and add it to the [repo secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) so Github Actions can reach Heroku. `heroku authorizations:create -d "Github Actions" -s write-protected`
+1. [Generate an auth token for Github](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-authentication-to-github#authenticating-with-the-api) and add it as [an environment variable](https://devcenter.heroku.com/articles/config-vars) so Heroku can trigger Github Actions
```

Comment:
Heroku auth token is easy to generate and lives forever. Adding it to Github settings is safe as well.
The Github auth token is currently a personal auth token of mine (Ed) with an expiration in a week or so. That could be a long-lived token, but it also required authing the token with JumpCloud. Not sure how long that lives for. PLUS, you then add that token to Heroku and all devs can see it, so not as safe.

I did look into a solution for that. Github Actions DOES generate a temporary app token while the workflows are running and I was able to pass that to Heroku using the Heroku CLI (same spot where we turn maintenance mode on). Problem is that the token gets expired when the workflow finishes. Seems like the only way around that is to keep the workflow alive....so then we're back to sleeping for 5min.